export_base_zola/config.toml

```diff
@@ -7,15 +7,17 @@ description = "Example blog description"
 # Whether to automatically compile all Sass files in the sass directory
 compile_sass = false
 
+generate_feed = true
+feed_filename = "rss.xml"
+feed_limit = 20
+
 # Whether to do syntax highlighting
 # Theme can be customised by setting the `highlight_theme` variable to a theme supported by Zola
+[markdown]
```

Comment:
@ratsclub what about this change? Does it mean to highlight code on markdown files?

Reply:
This means that it'll highlight code inside markdown files as far as I know. Without adding this section I couldn't get any highlighting.

Reply:
Oh, I see. Interesting. 👍